clusterloader2/pkg/test/simpletestexecutor.go

```diff
@@ -171,7 +176,16 @@ func (ste *simpleTestExecutor) ExecutePhase(ctx Context, phase *api.Phase) []err
 // ExecuteObject executes single test object operation based on provided object configuration.
 func (ste *simpleTestExecutor) ExecuteObject(ctx Context, object *api.Object, namespace string, replicaIndex int32, operation OperationType) []error {
 	var errList []error
-	template, err := ctx.GetTemplateProvider().GetTemplate(object.ObjectTemplatePath)
+	objName := fmt.Sprintf("%v-%d", object.Basename, replicaIndex)
+	var mapping map[string]interface{}
+	if object.Mapping == nil {
+		mapping = make(map[string]interface{})
+	} else {
+		mapping = object.Mapping
+	}
+	mapping[basenamePlaceholder] = objName
+	mapping[indexPlaceholder] = replicaIndex
+	template, err := ctx.GetTemplateProvider().GetTemplate(object.ObjectTemplatePath, mapping)
```

Comment:
Again here, I don't think you should be naming this variable 'template'. It's not a template here anymore iiuc, but the final object itself.